src/Serialisation/Serialisation.ts

```diff
@@ -0,0 +1,54 @@
+import { OMEOrder, SignedOrderData } from '../Types/types';
+
+/**
+ * Serialises an order from its raw signed format to a format supported by the OME
+ * @param web3 an instance of web3
+ * @param signedOrder raw signed order given by the signOrder function in this package
+ * @returns an order to be sent to the OME
+ */
+const orderToOMEOrder:(web3: any, signedOrder: SignedOrderData) => OMEOrder = (web3, signedOrder) => {
+    return {
+        id: "123",
```

Comment:
Does it matter that this never changes

Reply:
yeah id is redundant atm, we were planning on baking it into the OME as the hash of the order, however the OME currently still accepts an order id. I'll make a not to update the OME and remove this.